app-tests/cra-dashboard/package.json

```diff
@@ -29,6 +29,6 @@
     "@espressive/legacy-css": "latest",
     "react": "link:../node_modules/react",
     "react-dom": "link:../node_modules/react-dom",
-    "react-scripts": "4.0.3"
+    "react-scripts": "5.0.0"
```

Comment:
@brianespinosa, @rociobaena here are the highlights on react-scripts@5.0.0:

- webpack 5
- Jest 27
- **ESLint 8**
- PostCSS 8
- Fast Refresh improvements and bug fixes
- Support for Tailwind
- Improved package manager detection
- Unpinned all dependencies for better compatibility with other tools
- **Dropped support for Node 10 and 12**

I would like to bring to your attention the ones in **bold**:

**ESLint 8**
We might be ok to update as everything is configured by CRA, but still, we'll need to validate if everything works as expected.

**Dropped support for Node 10 and 12**
[We are locked to node v12](https://github.com/Espressive/cascara/blob/main/package.json#L134), we might not be able to bump if we want to keep it so.

I am doing some explorations on this specific bump, I'll report back my findings.